crates/turborepo-lib/src/config.rs

```diff
@@ -0,0 +1,134 @@
+use std::path::{Path, PathBuf};
+
+use anyhow::{Context, Result};
+use config::{Config, Environment};
+#[cfg(not(windows))]
+use dirs_next::config_dir;
```

Comment:
I used `directories` when writing the rust-go-rust sandwich proof of concept, will use `directories-next` (which depends on `dirs-next`) instead.